diracx-routers/src/diracx/routers/auth/management.py

```diff
@@ -74,7 +73,8 @@ async def revoke_refresh_token(
         subject = None
 
     try:
-        await revoke_refresh_token_bl(auth_db, subject, UUID(jti, version=4))
+        token_information = read_token(refresh_token, settings)
+        await revoke_refresh_token_bl(auth_db, subject, token_information["jti"])
```

Comment:
```suggestion
        await revoke_refresh_token_bl(auth_db, settings, subject, refresh_tokens)
```

Could you use `read_token` from `diracx-logic`? To keep all the logic at the same place.